src/sgame/sg_bot_util.cpp

```diff
@@ -1578,7 +1579,7 @@ void BotFireWeaponAI( gentity_t *self )
 		case WP_ALEVEL0:
 			break; //auto hit
 		case WP_ALEVEL1:
-			if ( distance < LEVEL1_CLAW_RANGE )
+			if ( distance < LEVEL1_CLAW_RANGE && readyFire )
```

Comment:
Why is this used only for level1? Also as a non-refactoring change it should be in a separate commit. `git add -p` is good.